.github/actions/restore-node/action.yml

```diff
@@ -185,8 +185,9 @@ runs:
       uses: pyTooling/Actions/with-post-step@v0.4.5
       with:
         main: |
-          bash "$SRC/.github/actions/restore-node/check-git-status.sh" "$SRC"
+          bash "$SRC/.github/actions/restore-node/check-git-status.sh" "$SRC" "$IGNORE_ENDO_BRANCH"
```

Comment:
_nitpick_

`IGNORE_ENDO_BRANCH` is not quite descriptive of what it's doing and it might be harder for someone to figure out what it does e.g. 6 months from now

Consider something like ignore dirty yarn lock

Reply:
Agreed! I've made the update.